lib/install/bootstrap/install.rb

```diff
@@ -3,10 +3,12 @@
    "app/assets/stylesheets/application.bootstrap.scss"
 run "yarn add sass bootstrap bootstrap-icons @popperjs/core"
 
-inject_into_file "config/initializers/assets.rb", after: /.*Rails.application.config.assets.paths.*\n/ do
-  <<~RUBY
-    Rails.application.config.assets.paths << Rails.root.join("node_modules/bootstrap-icons/font")
-  RUBY
+if Rails.root.join('config/initializers/assets.rb').exist?
+  inject_into_file 'config/initializers/assets.rb', after: /.*Rails.application.config.assets.paths.*\n/ do
+    <<~RUBY
+      Rails.application.config.assets.paths << Rails.root.join("node_modules/bootstrap-icons/font")
+    RUBY
+  end
```

Comment:
Maybe if it doesn't exist, we can do the same as [below](https://github.com/rails/cssbundling-rails/pull/82/files#diff-6a4335027393c4f283e26e55b5eb99d15469611c3ad6373a32cb4ab28e154ae1L16) and output what to do ?

Reply:
What do you think about

`Add Rails.application.config.assets.paths << Rails.root.join("node_modules/bootstrap-icons/font") in your assets configuration file (by convention in 'config/initializers/assets.rb')`

Reply:
Done

Reply:
And why just not create this file `config/initializers/assets.rb`, if it not exists?